en/config_vtol/vtol_duo_tailsitter_config.md

```diff
@@ -0,0 +1,94 @@
+The switch can also be assigned using the parameter [RC_MAP_TRANS_SW](../advanced_config/parameter_reference.md#RC_MAP_TRANS_SW).
+
+The switch in the off-position means that you are flying in multicopter mode.
+
+## Duo Tailsitter MC Mode Tuning Guide
+
+MC-mode tuning is dificult because some axes depend on control surfaces, while others depend on the motors.
+
+:::warning
+Always disable [MC_AIRMODE](../advanced_config/parameter_reference.md#MC_AIRMODE) when tuning a vehicle.
+:::
+
+As a sanity check, we start off by setting the following gain parameters:
+
+- Roll rate control:
+  - [MC_ROLLRATE_P](../advanced_config/parameter_reference.md#MC_ROLLRATE_P) = [MC_ROLLRATE_I](../advanced_config/parameter_reference.md#MC_ROLLRATE_I) = [MC_ROLLRATE_D](../advanced_config/parameter_reference.md#MC_ROLLRATE_D) = `0`, 
+  - [MC_ROLLRATE_K](../advanced_config/parameter_reference.md#MC_ROLLRATE_K) = `1`
+- Pitch rate control
+  - [MC_PITCHRATE_P](../advanced_config/parameter_reference.md#MC_PITCHRATE_P) = [MC_PITCHRATE_I](../advanced_config/parameter_reference.md#MC_PITCHRATE_I) = [MC_PITCHRATE_D](../advanced_config/parameter_reference.md#MC_PITCHRATE_D) = `0`
+  - [MC_PITCHRATE_K](../advanced_config/parameter_reference.md#MC_PITCHRATE_K) = `1`
+- Yaw rate control
+  - [MC_YAWRATE_P](../advanced_config/parameter_reference.md#MC_YAWRATE_P) = [MC_YAWRATE_I](../advanced_config/parameter_reference.md#MC_YAWRATE_I) = [MC_YAWRATE_D](../advanced_config/parameter_reference.md#MC_YAWRATE_D) = `0`
+  - [MC_YAWRATE_K](../advanced_config/parameter_reference.md#MC_YAWRATE_K) = `1`
+- Angle control
+  - [MC_ROLL_P](../advanced_config/parameter_reference.md#MC_ROLL_P) = [MC_PITCH_P](../advanced_config/parameter_reference.md#MC_PITCH_P) = [MC_YAW_P](../advanced_config/parameter_reference.md#MC_YAW_P) = `0`
+
+The expected behaviour is that when you arm in [Stabilized mode](../flight_modes/manual_stabilized_mc.md), both the motors should spin, control surfaces should be centred, and no effect of manual RC input using the transmistter is observed.
+
+:::note IF you can do so safely!
+Gently pick up the vehicle by hand and try tilting it in different axes.
+You should feel no countering torque from the motors or control surfaces.
+:::
+
+Once this sanity check is complete, we move on to slowly increasing the gains.
+
+### Rate Gains
+
+- Increase [MC_ROLLRATE_P](../advanced_config/parameter_reference.md#MC_ROLLRATE_P) by 0.1.
+  Arm and lift the vehicle and displace it in the roll axis.
+  You should feel a countering torque as the tailsitter tries to reorient itself.
+  This is still a very manual process, and there is not much science to it.
+  Increase the proportional gain until you can feel sustained oscillations when you displace it.
+  Then decrease the gain to 1/2 of that value.
```

Comment:
So say I increase my rate up to 0.6 for my and I can feel oscillations, you're saying the value I set is 0.3?

Reply:
Yep.